README.md

```diff
@@ -2,7 +2,7 @@
 ![Integration Tests](https://github.com/Layr-Labs/eigenda/actions/workflows/integration-tests.yml/badge.svg)
 ![Linter](https://github.com/Layr-Labs/eigenda/actions/workflows/golangci-lint.yml/badge.svg)
 ![Contracts](https://github.com/Layr-Labs/eigenda/actions/workflows/test-contracts.yml/badge.svg)
-![Go Coverage](https://github.com/Layr-Labs/eigenda/wiki/coverage.svg)
+[![codecov](https://codecov.io/github/Layr-Labs/eigenda/graph/badge.svg?token=EKLGVKW1VN)](https://codecov.io/github/Layr-Labs/eigenda)
```

Comment:
which coverage will this show? we output two separate ones right? for littdb and the other tests